recipes/deepspeed/bld.bat

```diff
@@ -0,0 +1,13 @@
+@echo On
```

Comment:
Since you are skipping windows is this file useful to you? have you tested it locally?

Reply:
ah nah, I just forgot about pytorch not being available on windows. I will delete it.